streaming/base/dataset.py

```diff
@@ -109,7 +109,7 @@ def exit(self) -> None:
             elif self._state == _IterState.EXITED:
                 return
             else:
-                raise RuntimeError(f'Invalid _IterState: {self._state}')
+                raise RuntimeError(f'Invalid Iterator state: {self._state}')
```

Comment:
This is technically an invalid _IterState enum. Although we will never get here in a sane world...

Reply:
Updated in https://github.com/mosaicml/streaming/pull/298/commits/ffb9eb9140df12f9f855fc02b82bf98c60ba776b.